src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,39 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.servise.DriverService;
+
 public class Main {
+    private static final Injector injector =
+            Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        //added driver to DB
+        Driver driverFirst = new Driver("Rita", "B1111");
+        Driver driverSecond = new Driver("Oscar", "C1112");
+        Driver driverThird = new Driver("Moony", "A01");
+        Driver driverForth = new Driver("Galo", "D07");
+        DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+        driverService.create(driverFirst);
+        driverService.create(driverSecond);
+        driverService.create(driverThird);
+        Driver createdDriverForth = driverService.create(driverForth);
```

Comment:
just use the same driverFourth, in your `create` method  you return the driver object which you got as an argument, but with `id` set